src/core/services/gesture/gesture.js

```diff
@@ -243,6 +244,18 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
         horizontal: true,
         cancelMultiplier: 1.5
       },
+      onSetup: function(element, opts) {
+        if (touchActionProperty) {
+          // We check for horizontal to be false, because otherwise we would overwrite the default opts.
+          this.oldTouchAction = element[0].style[touchActionProperty];
+          element[0].style[touchActionProperty] = opts.horizontal === false ? 'pan-y' : 'pan-x';
+        }
+      },
+      onCleanup: function(element) {
+        if (this.oldTouchAction) {
+          element[0].style[touchActionProperty] = this.oldTouchAction;
```

Comment:
There's also some overhead when you use jqLite functions.  Most of the time it doesn't matter, but it's good to err on the side of performance in cases like there unless the `css` method provides some benefit.